assets/src/components/AvatarModal.js

```diff
@@ -37,12 +37,13 @@ const styles = theme => ({
 })
 
 function AvatarModal (props) {
+  let globals = myla_globals
```

Comment:
Hi @vjcao, just one suggestion. Instead of accessing `myla_globals` from within `AvatarModal`, could you please add the `help_url` property to the frozen `user` object in `App`? 

Something like this:
```js
const user = Object.freeze({
  username: myla_globals.username,
  admin: myla_globals.is_superuser,
  enrolledCourses,
  isSuperuser: myla_globals.is_superuser,
  isLoggedIn: !!myla_globals.username,
  helpURL: myla_globals.help_url
})
```

Since the `user` object is passed to `AvatarModal` you'll be able to access the helpURL.

The reason I'm asking for this change is because I'm trying to reduce the number of components that directly access `myla_globals`. Since objects in JavaScript are passed by reference, there's a chance that some component could change some property in `myla_globals`, which could then cause widespread problems throughout the application, and it would be pretty tricky to debug as well.

Reply:
Thanks! I will change this.